docs/how-to-guides/new-user-guides/authentication-permissions-and-global-configuration/pod-security-standards.md

```diff
@@ -19,4 +19,22 @@ title: Pod Security Standards
 	* That updating a template will affect live clusters
 	* Who can edit. Including a template management role workaround with it's own warning/caveat.
 * How to upgrade
-	* Ensure that a replacement is in place before you upgrade and PSPs are off
\ No newline at end of file
+	* Ensure that a replacement is in place before you upgrade and PSPs are off
```

Comment:
```suggestion
	:::important
	
Make sure that you've set up a replacement and that Pod Security Policies (PSPs) are off before you try to upgrade.
	
	:::
```

I rephrased but we should specify what is being replaced (a replacement x?)

Reply:
I would let @jiaqiluo make the wider changes about how to upgrade and what things to do. 

This statement is specifically about security, so @pjbgf would you care to add a suggestion for a warning. 

Also this whole section is a "todo" i made for big things we need to document, so it all needs to be replaced soon.

Reply:
Feel free to embellish it:

Make sure that all [Pod Security Policies were migrated], which should result in equivalent security controls still being enforced (e.g. [Pod Security Admission], [Gatekeeper], [Kyverno]) and all `PodSecurityPolicy` objects removed from the cluster.
The new policy enforcement must be added **before** the `PodSecurityPolicy` objects are removed, or risk creating a window of opportunity for privilege escalation within the cluster.

[Pod Security Policies were migrated]: https://kubernetes.io/docs/tasks/configure-pod-container/migrate-from-psp/
[Pod Security Admission]: https://kubernetes.io/docs/concepts/security/pod-security-admission/
[Gatekeeper]: https://github.com/open-policy-agent/gatekeeper
[Kyverno]: https://github.com/kyverno/kyverno/